app/controllers/projects_controller.rb

```diff
@@ -59,7 +59,7 @@ def update
   private def project_params
     params.require(:project).permit(
       :urn,
-      :trust_ukprn,
+      :incomming_trust_ukprn,
```

Comment:
Had to double check, but incoming is with one "m" sorry 😬 . Hopefully just a find a replace?

Reply:
No probs!